bin/warmup_stats

```diff
@@ -288,8 +288,14 @@ def main(options):
     if options.output_plots:
         info('Generating PDF plots.')
         input_files = [bm.krun_filename_changepoints for bm in benchmarks]
+        window = benchmarks[0].window
+        if len(benchmarks) > 1:
+            for bm in benchmarks:
+                assert bm.window == window, 'File %s has window size %d, expected %d' \
```

Comment:
Since this is a check of the user's input, we shouldn't use `assert`, as they can be optimised away. We need to use something like `sys.stderr.write(...); sys.exit(1)`. Sorry :/

Reply:
I'm not sure this is a check of user input, we calculate the window size as a percentage of the number of iterations in each pexec:

```
class BenchmarkFile(object):
    ...
    def mark_outliers(self):
        self.window = int(self.iterations * DEFAULT_WINDOW_RATIO)
```

Does that make more sense?

Reply:
Maybe I'm confused, but we seem to be comparing the window's input and giving the error in terms of the user's file?

Reply:
That is true, let me write something clearer.